.github/workflows/ci.yml

```diff
@@ -241,12 +237,11 @@ jobs:
       - name: "Send a notification to Slack"
         # ENVs cannot be used directly in job.if. This is a workaround to check
         # if SLACK_WEBHOOK_URL is present.
-        if: "env.SLACK_WEBHOOK_URL != ''"
+        if: "${{ env.SLACK_WEBHOOK_URL != '' }}"
         uses: "slackapi/slack-github-action@v1.17.0"
         with:
           payload: |
             {
-              "text": "${{ env.SLACK_MESSAGE }}",
```

Comment:
Why remove this?

Reply:
It's a little further below in the text under the blocks. This is copied from nautobot-app-golden-config:ltm-1.6